### ISSUE TITLE
Implement the v2 protocol specification

### ISSUE THREAD

**Issue by eordano** [webtorrent/webtorrent#1117]
Opening this up for tracking the development. The protocol enhancement is still a work in progress.

Related discussions:
https://github.com/bittorrent/bittorrent.org/issues/58
https://github.com/bittorrent/bittorrent.org/pull/59

**Comment by eordano** [comment]
@feross It looks like the proposal has been merged! https://github.com/bittorrent/bittorrent.org/pull/59

Do you need any help with this?

**Comment by feross** [comment]
@eordano Yep, I'd like to support BitTorrent v2. I haven't had time to look at it in-depth yet.

@eordano If you have time, I'd love to see a PR for this. Also, happy to brainstorm how this might be implemented in this issue.

**Comment by resession** [comment]
any update on this?

**Comment by resession** [comment]
so by v2, you mean BEP 52

https://www.bittorrent.org/beps/bep_0052.html

right?
